src/ui/application-window.vala

```diff
@@ -251,6 +254,23 @@ namespace Peek.Ui {
       record_button.set_label (_("Record as %s").printf (format_name));
       pop_format.hide ();
     }
+    
+    private void update_time () {
+      int seconds=0;
```

Comment:
Old habbits :p Also two lines lower

Reply:
Oh yeah It is automatic.. like autopilot :)

Reply:
Please update the spacing around `=` and `+=` below, than this PR looks great code wise.